app/controllers/monsters_controller.rb

```diff
@@ -0,0 +1,6 @@
+class MonstersController < ApplicationController
+
+	def index
+		@monsters = Monster.all
+	end
```

Comment:
Let's watch indentation here so it is consistent in all files throughout the group

Reply:
Done